webtask.json

```diff
@@ -25,7 +25,7 @@
     "onInstallPath": "/.extensions/on-install",
     "onUpdatePath": "/.extensions/on-install",
     "scopes":
-      "read:connections read:users read:rules create:rules update:rules delete:rules delete:clients"
+      "read:connections read:users read:rules create:rules update:rules delete:rules delete:clients read:custom_domains"
```

Comment:
Did this correctly update the existing client?

Reply:
We were able to grab the custom domains by adding that scope

Reply:
Was wondering what if upgrading an existing extension was tested along with created a new extension. There was a bit of ambiguity in update process for extensions when discussing earlier this week in sync.

Reply:
```
    "onInstallPath": "/.extensions/on-install",
    "onUpdatePath": "/.extensions/on-install",
 ```

Reply:
Those file path's don't live in the repo